tests/integration/records/views/test_views_journals.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+def test_journals_application_json_get(api_client, db, create_record_factory):
+    record = create_record_factory("jou", with_indexing=True)
```

Comment:
can we just reuse the create_record now?

Reply:
I didn't touch old tests for views except moving them to separate folder. If you wish I can change this tests to use create_record.